examples/cronjobs-analytics-waspleau/src/client/WaspSourceHeader.jsx

```diff
@@ -1,4 +1,4 @@
-import waspLogo from './waspLogo.png'
+import waspLogo from './waspLogo.svg'
```

Comment:
All these changes you did in examples -> they are not really needed. It doesn't really matter if they use waspLogo.png, it is just a file generated at start. So I would revert these changes.

Reply:
Also, most of the example you corrected only half way -> you changed the import, but didn't swap .png file with .svg file. You did that in only one example. So I believe only one example works correctly.